rancher/rancher.yaml

```diff
@@ -0,0 +1,91 @@
+
+---
+kind: Namespace
+apiVersion: v1
+metadata:
+  name: cattle-system
+  labels:
+    name: cattle-system
+
+---
+kind: ServiceAccount
+apiVersion: v1
+metadata:
+    name: cattle-admin
+    namespace: cattle-system
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+    name: cattle-crb
+    namespace: cattle-system
+subjects:
+  - kind: ServiceAccount
+    name: cattle-admin
+    namespace: cattle-system
+roleRef:
+    kind: ClusterRole
+    name: cluster-admin
+    apiGroup: rbac.authorization.k8s.io
+
+---
+kind: Deployment
+apiVersion: extensions/v1beta1
+metadata:
+    namespace: cattle-system
+    name: cattle
+spec:
+    replicas: 1
+    template:
+      metadata:
+        labels:
+          app: cattle
+      spec:
+        serviceAccountName: cattle-admin
+        containers:
+        - image: rancher/rancher:stable
```

Comment:
We always prefer to lock this down to a particular version - so if Rancher (in this case) breaks something upstream at least we have a known good version here.

Reply:
ok Let me give the Latest GA version 2.3

Reply:
👍